### PR TITLE
en: improve format and fix errors in existing files

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

Fix some obvious errors in the existing files.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by ran-huang** [comment]
I ran a grammar check on **25 files** and fixed some simple errors.

There are still about **40 files I haven't checked** (and I'm sure they are not error-free😅), and it might take a week or two to completely rid our docs of those errors. However, keeping this PR open might bring conflicts to newly opened PRs, so I think it'd be better that we just review and merge the first batch.

I'll come back to the rest 40 files when I have time.

**Comment by DanielZhangQD** [comment]
> I ran a grammar check on **25 files** and fixed some simple errors.
> 
> There are still about **40 files I haven't checked** (and I'm sure they are not error-free😅), and it might take a week or two to completely rid our docs of those errors. However, keeping this PR open might bring conflicts to newly opened PRs, so I think it'd be better that we just review and merge the first batch.
> 
> I'll come back to the rest 40 files when I have time.

Thanks a lot!

File: en/access-dashboard.md [hunk 1]
```diff
@@ -156,9 +156,9 @@ type: kubernetes.io/tls
 
 After Ingress is deployed, visit <https://{host}/dashboard> to access TiDB Dashboard.
 
-## Update TiDB cluster
+## Update the TiDB cluster
 
-If you enable quick access to TiDB Dashboard by updating an existing TiDB cluster, you need update the following two configurations:
+If you enable quick access to TiDB Dashboard by updating an existing TiDB cluster, update the following two configurations:
```
Comment (lilin90) [inline en/access-dashboard.md:161]:
```suggestion
To enable quick access to TiDB Dashboard by updating an existing TiDB cluster, update the following two configurations:
```

File: en/deploy-tidb-operator.md [hunk 2]
```diff
@@ -18,19 +18,19 @@ Before deploying TiDB Operator, make sure the following items are installed on y
 * [RBAC](https://kubernetes.io/docs/admin/authorization/rbac) enabled (optional)
 * [Helm](https://helm.sh) version >= 2.11.0 && < 3.0.0 && != [2.16.4](https://github.com/helm/helm/issues/7797)
 
-## Deploy Kubernetes cluster
+### Deploy the Kubernetes cluster
```
Comment (lilin90) [inline en/deploy-tidb-operator.md:21]: It seems that the 2nd-level heading is expected here.
```suggestion
## Deploy the Kubernetes cluster
```

File: en/deploy-tidb-operator.md [hunk 3]
```diff
@@ -18,19 +18,19 @@ Before deploying TiDB Operator, make sure the following items are installed on y
...
 
-TiDB Operator uses [Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to persist the data of TiDB cluster (including the database, monitoring data, and backup data), so the Kubernetes cluster must provide at least one kind of persistent volume. For better performance, it is recommended to use local SSD disk as the volumes. Follow [this step](#configure-local-persistent-volume) to provision local persistent volumes.
+TiDB Operator uses [Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) to persist the data of TiDB cluster (including the database, monitoring data, and backup data), so the Kubernetes cluster must provide at least one kind of persistent volumes. For better performance, it is recommended to use local SSD disks as the volumes. Follow [this step](#configure-local-persistent-volumes) to provision local persistent volumes.
 
 It is recommended to enable [RBAC](https://kubernetes.io/docs/admin/authorization/rbac) in the Kubernetes cluster.
 
-## Install Helm
+### Install Helm
```
Comment (lilin90) [inline en/deploy-tidb-operator.md:29]:
```suggestion
## Install Helm
```

**Pull request merged**